docs/src/pages/theming/svelteui-provider.md

```diff
@@ -44,7 +44,7 @@ For SvelteUI to work properly, you need to set up the SvelteUIProvider at the to
 </script>
 
 <SvelteUIProvider>
-	<YourApp />
```

Comment:
This isn't a typo, so you will need to change this back :)

Reply:
I haven't messed with the package.json, the package-**lock**.json was probably updated by some npm run script when setting up dev. 

Could you elaborate on how it is not a typo? Adding the entire App as a component to __layouts.svelte would duplicate it for every route. So I don't see how it would be possible to make use of SvelteKits filesystem-based router with the current approach.

Reply:
It's not saying that you should put your entire app there, it's more of an "expression."

Reply:
Mm okay. What then is the point of this "expression"? The current solution does not allow one to just copy the text whereby it "just works". It's confusing at best and hinders the adoption of this library. Besides, you are literally using my version in other parts of the docs: https://www.svelteui.org/theming/ssr

Reply:
Fair point, It's nothing that is too pressing. So I can merge it as is right now.